### ISSUE TITLE
The data augmentation didn't apply in the model training?

### ISSUE THREAD

**Issue by PinRuei** [Nithyashree-2022/VGG-19-for-Rock-Paper-and-Scissors-classification#1]
In your code, the data augmentation output is **training_set**, but in the model fitting, you just use **train_x** as the training dataset. And when I want to use the training_set, the program will got  the error " cannot identify image file <_io.BytesIO object at 0x7f159d9f9b90> ".

history = model.fit(
  training_set,
  validation_data=(val_set),
  epochs=10,
  callbacks=[early_stop],
  batch_size=32,shuffle=True)

**Comment by Nithyashree-2022** [comment]
Yes in this model.fit() it accepts only train_x as it is a NumPy array of representation of the images of our dataset. The function cannot accept training_set as its parameter as this is of a different type. That data augmentation code is mainly to get the labels of the images traversing every directory in our dataset i.e., to get train_y(get labels of every image of our dataset).

**Comment by PinRuei** [comment]
Thanks for your reply. 
In your opinion, the data augmentation code is to get the labels, so it means if I change the code as below that will make the label(train_y) no match the data(train_x)? 

train_datagen = ImageDataGenerator(rescale=1./255,shear_range=0.2,zoom_range=0.2,horizontal_flip=True)

Since I want to apply data augmentation in the training dataset, do you know any suggestion to change the code?
